script/arbitrum-mainnet/deployer.sol

```diff
@@ -120,8 +120,8 @@ library ArbitrumMainnetDeployer {
 
         DeploymentHelper.PairConfig memory USDTWETHPairConfig = DeploymentHelper.PairConfig({
             name: "USDT/WETH",
-            durations: singleDuration,
-            ltvs: singleLTV,
+            durations: allDurations,
```

Comment:
what is the reason for some pairs using all durations / ltvs and some single? (like WBTC below?)

Reply:
initially when i talked about it with jpaul some pairs werent gonna use all durations available, but i ended up adding all durations to all cause tests dont work cause twap cant catch up to price movement

honestly the whole duration side of this should be changed cause its now a range on confighub, can do in another pr